00. Packages to install/1. Genestack python client/README.md

```diff
@@ -28,26 +30,29 @@ python3 -m pip install \
 
    You will recieve an email with a link to your token – if using the Public account, you may need to visit the mailcatcher to obtain this link.
 
-3. Set up your account with the Genestack Python client from a console with -H and provide an ODM host url in the <ODM>
-   ```genestack-user-setup -H https://<ODM>/frontend```
+3. Set up your account with the Genestack Python client from a console
+   ```shell
+   genestack-user-setup 
+   ```
 
-4. Then select authentication method by token (1) and input the content of the token you received in step 2:
-   ``` 
+4. Type ‘add’ to enter a new user, enter an alias for the user.
```

Comment:
After this step we should also add host name
Previously in the old instruction we had info:

> Enter the host name, which should be of the format: https://domain_name/frontend